unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridItemComponentView.cs

```diff
@@ -85,6 +95,23 @@ public override void RefreshControl()
             nftBackground.sprite = rarityNftBackgrounds.GetRarityImage(nftRarity);
             categoryBackground.color = nftTypesIcons.GetColor(nftRarity);
             categoryImage.sprite = nftTypesIcons.GetTypeImage(model.Category);
+
+            if (model.IsCompatibleWithBodyShape)
+                image.ImageComponent.material = null;
+            else
+            {
+                image.ImageComponent.material = grayScaleMaterial;
+                image.ImageComponent.SetMaterialDirty();
```

Comment:
it's redundant: material is already set dirty when you set `material` to `Image`

Reply:
Done